lib/source-check.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const fs = require('fs')
+
+function checkForTranspiledCode (filename) {
+  const readFile = fs.readFileSync(filename, 'utf8')
+  const regex = /function\s+(\w+)/g
+  let matchedObj
+  let isTranspiled = false
+
+  // Check for a source map
+  if (readFile.includes('//# sourceMappingURL=')) {
+    isTranspiled = true
+  } else {
+    // Loop through results and check length of fn name
+    while ((matchedObj = regex.exec(readFile)) !== null) {
+      if (matchedObj[1].length < 3) {
+        isTranspiled = true
+      } else {
+        isTranspiled = false
+      }
```

Comment:
this if/else will now effectively treat the last function name in the source code as authoritative. flipping it to false if a long fn name is found is a good idea; then we should initialise `isTranspiled = true` right before entering the loop

Reply:
Okay, I've removed the else and reversed the check. Hopefully this is the perfect version.